MCDTool.egg-info/PKG-INFO

```diff
@@ -0,0 +1,14 @@
+Metadata-Version: 2.1
+Name: MCDTool
+Version: 0.0.1
+Summary: A probabilistic Multi Criteria Decision Analysis
+Home-page: UNKNOWN
+Author: Flaminia Catalli - wetransform GmbH & Matteo Spada - Zurich University of Applied Sciences
+Author-email: fc@wetransform.to, spaa@zhaw.ch
+License: UNKNOWN
+Platform: UNKNOWN
+Requires-Python: >=3.9
+License-File: LICENSE
+
+UNKNOWN
```

Comment:
What is this `UNKNOWN` for?

Reply:
I think it might happen because I am omitting the "Classifiers" section in the setup.py (see https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#classifiers) but I am not sure.